src/components/search/__test__/SearchPage.test.tsx

```
@@ -0,0 +1,76 @@
+
+// Mock the child components to isolate the SearchPage component testing
+vi.mock('@/components/chat/LegalChatbot', () => ({
+    LegalChatbot: () => <div data-testid="legal-chatbot-mock">Legal Chatbot Mock</div>
+}));
+
+vi.mock('@/components/chat/ChatHistory', () => ({
+    ChatHistory: () => <div data-testid="chat-history-mock">Chat History Mock</div>
+}));
+
+vi.mock('@/components/ui/button', () => ({
+    Button: ({ children, ...props }: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)